django/contrib/postgres/search.py

```diff
@@ -142,10 +143,16 @@ def as_sql(self, compiler, connection):
         params = [self.value]
         if self.config:
             config_sql, config_params = compiler.compile(self.config)
-            template = 'plainto_tsquery({}::regconfig, %s)'.format(config_sql)
+            if self.phrase:
```

Comment:
I think this could be made more DRY:
```python
function = 'phraseto_tsquery' if self.phrase else 'plainto_tsquery'
if self.config:
    ...
else:
    ...
```